src/python/pants/notes/2.17.x.md

```diff
@@ -5,11 +5,25 @@
 ### Overall
 
 The [`scie-pants` launcher binary](https://github.com/pantsbuild/scie-pants) is now the recommended way to run Pants.
-Use of a `./pants` shell script in a repository is deprecated. Among other benefits, the `scie-pants` launcher is able
-to automatically download and maintain a Python distribution for running Pants. It also allows you to run `pants`
-from any directory within a repository and not just the build root. See the [Pants  installation
+Use of a `./pants` shell script in a repository is deprecated.
+
+The `scie-pants` launcher has numerous benefits, such as:
+
+* automatically download and maintain a Python distribution for running Pants
+* run `pants` from any directory within a repository and not just the build root
+* far less overhead when starting commands (up to 4-5×)
+
+See the [Pants  installation
 instructions](https://www.pantsbuild.org/docs/installation) for additional information on how to migrate to `scie-pants.`
 
+Related to this, pants' distribution model is changing, and 2.17 will be the last version that includes the universal (cross-platform) `pants` PEX on the GitHub Release. For future releases, smaller per-platform PEXes will be attached instead.
```

Comment:
Should we mention 2.17 will be the last on PyPI as well?

Reply:
Done!